### PR TITLE
Regression tests update for new boost serialisation change

### PR DESCRIPTION
**Pull request by ggovi**
Condition Database:
Update for regression test to check compatibility of boost serialisation across versions

Reader list is empty, waiting for the (10_4?) reference release
SELF is the only readers ( no failure anymore )
Writer list has been cleaned up

### CODE REVIEW

**Comment by ggovi** [comment]
please test

**Comment by ggovi** [comment]
+1

File: CondCore/CondDB/test/condTestRegression.py [hunk 1]
```diff
@@ -9,13 +9,25 @@
...
+# Boost 1.67 [No reference release yet...]
+readers = {
           }
 
-writers = { 'CMSSW_9_0_1'        : [ ('slc6_amd64_gcc530', 'ref901-s6530.db') ],
-            'CMSSW_8_1_0'        : [ ('slc6_amd64_gcc530', 'ref810-s6530.db') ],
-            'CMSSW_8_0_26'       : [ ('slc6_amd64_gcc530', 'ref8026-s6530.db')],
-            'CMSSW_7_6_6'        : [ ('slc6_amd64_gcc493', 'ref766-s6493.db')],
+writers = { 'CMSSW_9_0_1'        : [ ('slc6_amd64_gcc630', 'ref901-s6630.db')],
+            'CMSSW_8_1_0'        : [ ('slc6_amd64_gcc530', 'ref750-s6530.db'),('slc6_amd64_gcc600', 'ref750-s600.db')],
```
Comment (fabiocos) [inline CondCore/CondDB/test/condTestRegression.py]: @ggovi for my understanding, why this change compared to the previous version where both 8_0_26 and 8_1_0 were appearing with gcc530?

File: CondCore/CondDB/test/condTestRegression.py [hunk 2]
```diff
@@ -9,13 +9,25 @@
...
+# Boost 1.57
+#readers = { 'CMSSW_7_5_0'        : ['slc6_amd64_gcc491'], 
+#          }
+
+# Boost 1.63
+#readers = { 'CMSSW_9_0_0'        : ['slc6_amd64_gcc530'], 
+#          }
+
+# Boost 1.67 [No reference release yet...]
+readers = {
```
Comment (fabiocos) [inline CondCore/CondDB/test/condTestRegression.py]: @ggovi so at present this just picks the current architecture of the used build, am I correct?

**Comment by fabiocos** [comment]
@ggovi could you please clarify about the couple of minor comments I had?

File: CondCore/CondDB/test/condTestRegression.py (hunk 1 above)
Comment (ggovi) [reply CondCore/CondDB/test/condTestRegression.py]: 901 and 810 makes more sense since they are respectively the first release built with gcc630 and gcc530. I honestly don't remember why we had the 8_0_26, I suspect was simply taken the release used to develop the test.

File: CondCore/CondDB/test/condTestRegression.py (hunk 2 above)
Comment (ggovi) [reply CondCore/CondDB/test/condTestRegression.py]: yes, there is no full release yet available as a reader, there will be one after a full 10_4 will be available.

**Comment by fabiocos** [comment]
+1
